GOVERNANCE.md

```diff
@@ -0,0 +1,49 @@
+# Project Governance (Very Minimal Governance model)
+
+This open source project is managed by a Steering Committee composed of the maintainers of this project. Maintainers are
+defined as individuals with full commit access to the project repositories.
```

Comment:
Can we add something to "full commit access" like [maintain-level permissions on a repository](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization).